src/request/body.rs

```diff
@@ -68,3 +73,76 @@ impl<T: Serialize> From<(String, T)> for GraphQLBody<T> {
         (query.as_str(), Some(variables)).into()
     }
 }
+
+#[derive(Debug, Clone)]
+pub struct MultipartFile {
+    pub(crate) bytes: Vec<u8>,
+    pub(crate) name_opt: Option<String>,
+    pub(crate) mime_type_opt: Option<String>,
+}
+
+impl MultipartFile {
+    pub fn new(bytes: Vec<u8>) -> Self {
+        Self {
+            bytes,
+            name_opt: None,
+            mime_type_opt: None,
+        }
+    }
+
+    pub fn with_name(self, name: impl Into<String>) -> Self {
+        Self {
+            name_opt: Some(name.into()),
+            ..self
+        }
+    }
+
+    pub fn with_mime_type(self, mime_type: impl Into<String>) -> Self {
+        Self {
+            mime_type_opt: Some(mime_type.into()),
+            ..self
+        }
+    }
+
+    pub(crate) fn into_part(self) -> PrimaBridgeResult<Part> {
```

Comment:
This could be a `TryInto`?

Reply:
I suppose we could, but it's not IDE or docs friendly (e.g. won't show up in autocomplete), so I would hesitate to implement it just for the sake of implementing a trait instead.

Reply:
I would say that this is the default way the transformation of a type to another type.. I don't get why is not docs friendly :sweat_smile:

Reply:
From<T> implementations don't show up in IDE autocompletion, so it's not immediately apparent that _this is how you are supposed to turn A to B_, one has to refer to the docs for that struct, which increases development time and confusion from implicit behaviour - I think From<T> implementations are more intended for when a type can be converted to _multiple_ different types.

Reply:
In this specific case, the most compelling reason not to implement `TryInto` is to avoid making the method public.